lib/lang/my_localizations.dart

```diff
@@ -46,4 +46,21 @@ class MyLocalizations {
         desc: 'Text on sign in button',
         examples: const {'service': 'Google'},
       );
+
+  String howToText(int index) {
```

Comment:
Rewrite this so the switch statement only sets the value of a string and then return a Message with a name, args and description (see above). This is important for generating translation files.